examples/terraform-submodules/gke-autopilot/module.tf

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Run:
+//  terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.30.0"]
+
+terraform {
```

Comment:
Are you going to have a following up PR to include the instruction about the usage or this Module? I assume that will be something like [this](https://agones.dev/site/docs/installation/terraform/gke/)?

Reply:
Yeah - it's going out in a broader set of doc updates when we work out how we want to structure it.